docs_src/text.ipynb

```diff
@@ -259,8 +259,8 @@
    "metadata": {},
```

Comment:
<p>I can add comments, this is amazing!!!!!!!!!!!!!!!!</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>